src/components/light.js

```diff
@@ -46,10 +46,37 @@ module.exports.Component = registerComponent('light', {
       // Light type has not changed. Update light.
       Object.keys(diffData).forEach(function (key) {
         var value = data[key];
-        if (['color', 'groundColor'].indexOf(key) !== -1) {
-          value = new THREE.Color(value);
+
+        switch (key) {
+          case 'color':
+            light[key] = new THREE.Color(value);
```

Comment:
You don't need to instanciate a new color. You can just set it `light.color.set(value)`.